modules/core/shared/src/main/scala/gem/config/StaticConfig.scala

```diff
@@ -15,12 +15,14 @@ import monocle.macros.Lenses
  */
 sealed abstract class StaticConfig {
   type I <: Instrument with Singleton
-  def instrument: I
+  val instrument: Instrument.Aux[I]
```

Comment:
Seems like that should remain a `def` if it isn't defined.

Reply:
Yeah I'll switch that back. An earlier attempt needed a stable reference.